time timestamp
uint16 slaveid
uint16 read_input_registers_count
uint16 read_input_registers_start_address
uint16[] read_input_registers
